create_bringup/package.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0"?>
-<package format="2">
+<package format="3">
```

Comment:
Why format 3? I think we can continue to use format 2.

Reply:
So that all package.xml files would be on the same version. Reverted.